var/spack/repos/builtin/packages/py-graphql-relay/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyGraphqlRelay(PythonPackage):
+    """Relay library for graphql-core."""
+
+    homepage = "https://github.com/graphql-python/graphql-relay-py"
+    pypi = "graphql-relay/graphql-relay-2.0.1.tar.gz"
+
+    maintainers("LydDeb")
+
+    version("2.0.1", sha256="870b6b5304123a38a0b215a79eace021acce5a466bf40cd39fa18cb8528afabb")
+
+    depends_on("python@3.4:3.8", type=("build", "run"), when="@2")
```

Comment:
Where do you see that newer versions of Python aren't supported? From testing?

Reply:
From commit message. Take a look at https://github.com/graphql-python/graphql-relay-py/commit/26036ef2a9da755052bec48b9449178b17652528

Reply:
That comment documents that 3.8 is supported but it doesn't say that 3.9 isn't. The versions documented in `classifiers` are just the ones that are tested. The package will likely work with newer versions of Python if you test them. I don't restrict Python unless I know that it won't work.